### ISSUE TITLE
Update Python Versions

### ISSUE THREAD

**Issue by askpatrickw** [BradenM/micropy-cli#109]
Before I submit a PR, wanted to run this by you.

Currently .python-version has
3.7.3
3.6.8

I proposed updating this to:
3.8.3
3.7.7
3.6.9

The Travis and GH-Actions in TOX.ini and would also need to be updated
```
[tox]
isolated_build = true
envlist = py38, py37, py36, flake8

[travis]
python =
    3.8: py38
    3.7: py37
    3.6: py36

[gh-actions]
python =
    3.8: py38
    3.7: py37
    3.6: py36
```

Not sure if any other places need updating for this OR if any other dependencies will need to be updated as well.

**Comment by BradenM** [comment]
I agree. Both the base python versions _and_ several of `micropy-cli`'s dependencies need to be updated. Will look into soon.

**Comment by BradenM** [comment]
Closed by #113
